pom.xml

```diff
@@ -95,6 +104,25 @@
 					<disabledTestInjection>true</disabledTestInjection>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>findbugs-maven-plugin</artifactId>
+				<version>${findbugs-maven-plugin.version}</version>
+				<configuration>
+					<failOnError>${maven.findbugs.failure.strict}</failOnError>
+					<xmlOutput>true</xmlOutput>
+					<findbugsXmlWithMessages>true</findbugsXmlWithMessages>
```

Comment:
findbugsXmlWithMessages does not exist anymore:
https://issues.jenkins-ci.org/browse/JENKINS-29489